CHANGELOG.md

```diff
@@ -2,6 +2,10 @@
 
 ## Version 4.1.1 (Under development)
 
+### AppCenter
+
+* **[Fix]** Fix SDK not sending events when application is packed to self-contained single-file executable.
```

Comment:
Is this fix needed only for .NET 5 self-contained apps? If so, it is worth mentioning that here and in the comment in `DeviceInformationHelper`.

Reply:
I suppose this also applies to versions prior to .NET 5 (at least that's what one can think looking at documentation), so I'd better keep it the way it currently is.